updating/understanding-openshift-update-duration.adoc

```diff
@@ -0,0 +1,47 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-update-duration"]
+= Understanding {product-title} update duration
+include::_attributes/common-attributes.adoc[]
+:context: openshift-update-duration
+
+toc::[]
+
+{product-title} update duration varies based on the deployment topology. This section helps you understand the factors that affect update duration and estimate how long the cluster update takes in your environment.
+
+== Prerequisites
+* You are familiar with xref:../architecture/architecture.adoc#architecture[OpenShift Container Platform architecture] and xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[OpenShift Container Platform updates].
+
+include::modules/update-duration-factors.adoc[leveloffset=+1]
+
+== Cluster update phases
+In {product-title}, the cluster update happens in two phases:
+
+* Cluster Version Operator (CVO) target update payload deployment
+* Machine Config Operator (MCO) node updates
+
+
+include::modules/update-duration-cvo.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../architecture/architecture-installation.adoc#update-service-overview_architecture-installation[Cluster Version Operator (CVO) overview]
+
+include::modules/update-duration-mco.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../post_installation_configuration/machine-configuration-tasks.adoc#machine-config-overview-post-install-machine-configuration-tasks[Machine config overview]
+* xref:../nodes/pods/nodes-pods-configuring.adoc#nodes-pods-configuring-pod-distruption-about_nodes-pods-configuring[Pod disruption budget]
+
+
+
+
+include::modules/update-duration-estimate-cluster-update-time.adoc[leveloffset=+1]
+
+include::modules/update-duration-rhel-nodes.adoc[leveloffset=+1]
+[role="_additional-resources"]
```

Comment:
I recommend adding a blank line in between the `include` line and the role definition line. In this case it does not seem to impact the rendering. But, I have seen situations where not having a blank line after a set of `include` lines can cause problems with the rendering.